cmd/translate/main_rest_integration_test.go

```diff
@@ -18,9 +19,12 @@ import (
 	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
 )
 
+// TODO Currently, we manually create requests for the REST API.
+// We could use a client generated from the OpenAPI specification to simplify testing and integration.
+
 // -------------Translation File-------------.
 
-func attachFile(text []byte, t *testing.T) (*bytes.Buffer, string, error) {
+func attachFile(text []byte, t *testing.T) (*bytes.Buffer, string) {
 	t.Helper()
 
 	body := &bytes.Buffer{}
```

Comment:
It's better to declare only and pass or return a reference to it.

```suggestion
	var body bytes.Buffer
```